compiler/rustc_privacy/src/lib.rs

```diff
@@ -1741,7 +1747,7 @@ impl SearchInterfaceForPrivateItemsVisitor<'tcx> {
                         self.visit(self.tcx.type_of(param.def_id));
                     }
                 }
-                GenericParamDefKind::Const => {
+                GenericParamDefKind::Const { .. } => {
```

Comment:
can you add a `FIXME(const_evaluatable_checked)` here? We may want to look inside of the const here